spec.bs

```diff
@@ -1686,16 +1683,26 @@ To <dfn>convert a modifier to a string</dfn> given a [=part/modifier=] |modifier
   1. Set |result|["{{URLPatternInit/hash}}"] to |hash|.
   1. Let |baseURL| be null.
   1. If |init|["{{URLPatternInit/baseURL}}"] is not null:
```

Comment:
I think this algorithm has a persistent, existing problem of conflating "not present" and null. This is worse now because you're doing the right thing and doing "exists" checks in your new spec text, but the spec text right below it is doing "null" checks.

Probably you should fix them all to be "exists" checks.